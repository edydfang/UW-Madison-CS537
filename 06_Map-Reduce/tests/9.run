./sort tests/1.txt tests/2.txt 2 1 1
