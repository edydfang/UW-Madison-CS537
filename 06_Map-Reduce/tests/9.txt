gun
twist
parachute
angel
abuse
breakin
adventure
worm
frog
rage
deficiency
snack
colleague
trip
refrigerator
experiment
soil
summit
kid
pioneer
conductor
knot
thesis
variant
bolt
branch
positive
fight
cellar
quit
volume
sensitive
choke
establish
halt
hit
cable
unlike
dangerous
experience
jewel
core
ego
pipe
home
healthy
launch
final
hypnothize
porter
penalty
despise
sanctuary
information
van
influence
paint
ancestor
refuse
instruction
satellite
jelly
program
pile
Venus
resort
global
jurisdiction
tiger
weight
fat
cap
stadium
performer
evolution
national
pick
lost
guerrilla
pneumonia
pursuit
simplicity
valid
create
philosophy
nap
rider
paragraph
lane
minister
tread
explosion
beg
arm
fruit
noble
tin
gas
gaspedal
slot
