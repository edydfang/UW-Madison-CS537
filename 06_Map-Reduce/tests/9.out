3344556677
3456346
34565363
523654
653465
75687785
